File: Python/testing.py
import math

n=12345
print(n)
digits=int(math.log(n,10))+1
print(digits)

a="abcde"
print(a.startswith("ab"))
print(a)
print(a[2])
print(type(a))
print(type(a[2]))

list=[]
list.append(1)
list.append(2)
list.append(3)
print(list)
list2=[]
list2.extend(list[:1])
print(list2)

File: Javascript/testing.js
n=12345
console.log(n)

digits=Math.log10(n)+1
console.log(Math.floor(digits))

a="abcde"
console.log(a)
console.log(a[2])
console.log(a.startsWith('ab'))

list=[]
list[0]=1
list[1]=2
list2={}
list2=[...list,21]//slice list??
console.log(list)
console.log(list2)